Twig/TranslationExtension.php

```diff
@@ -64,8 +64,19 @@ public function transchoiceWithDefault($message, $defaultMessage, $count, array
             $domain = 'messages';
         }
 
+        // If < sf2.6
+        if (!method_exists($this->translator, 'getCatalogue')) {
+            return $this->transchoiceWithDefaultLegacy($message, $defaultMessage, $count, $arguments, $domain, $locale);
+        }
+
         if (false == $this->translator->getCatalogue($locale)->defines($message, $domain)) {
-            return $this->translator->transChoice($defaultMessage, $count, array_merge(array('%count%' => $count), $arguments), $domain, $locale);
+            return $this->translator->transChoice(
+                $defaultMessage,
+                $count,
+                array_merge(array('%count%' => $count), $arguments),
+                $domain,
+                $locale
+            );
         }
```

Comment:
My only thought here - particularly with the large backlog of PRs is that we shouldn't change formatting at the same time as changing code. This could potentially make other merges more complicated.